tensorflow_probability/python/bijectors/real_nvp_test.py

```diff
@@ -97,6 +98,64 @@ def testBatchedBijectorWithMLPTransform(self):
       self.assertAllClose(x_, inverse_y_, rtol=1e-4, atol=0.)
       self.assertAllClose(ildj_, -fldj_, rtol=1e-6, atol=0.)
 
+  def testBijectorConditionKwargs(self):
+    batch_size = 3
+    x_ = np.linspace(-1.0, 1.0, (batch_size * 4 * 2)).astype(
+        np.float32).reshape((batch_size, 4 * 2))
+
+    conditions = {
+        "a": tf.random_normal((batch_size, 4), dtype=tf.float32),
+        "b": tf.random_normal((batch_size, 2), dtype=tf.float32),
+    }
+
+    def _condition_shift_and_log_scale_fn(x0, output_units, **condition_kwargs):
```

Comment:
fwiw, here you could replace condition_kwargs with just the named args, a and b:

```
def _condition_shift_and_log_scale_fn(x0, output_units, a, b):
   x = tf.concat([x0, a, b], axis=-1)
...
```

Reply:
Noted. I changed this and also the one below in the `RealNVPConstantShiftScaleTest` which didn't require `condition_kwargs` at all.

Thanks!